src/oneD/MultiNewton.cpp

```diff
@@ -205,13 +218,14 @@ int MultiNewton::dampStep(const double* x0, const double* step0,
 {
     // write header
     if (loglevel > 0 && writetitle) {
-        writelog("\n\nDamped Newton iteration:\n");
-        writeline('-', 65, false);
-
-        writelog("\n{}  {:>9s}   {:>9s}     {:>9s}   {:>9s}   {:>9s}  {:>5s} {:>5s}\n",
-                "m","F_damp","F_bound","log10(ss)",
-                "log10(s0)","log10(s1)","N_jac","Age");
-        writeline('-', 65);
+        string separator = fmt::format("  {:->80}", "");
```

Comment:
The default with of many terminals is 80 characters. This would create a line length that's slightly longer, correct? If so, I'd recommend reducing to 80 to prevent ugly line breaks.

Reply:
I was thinking about that. I wasn't sure, with modern monitors if the 80 character width is still a hard limit.

Reply:
I will try to compress the data down some more. I liked the tab over for the outputs to help with nesting the outputs to help visually separate the different parts of the solver proceedure.